pkg/sentry/fsimpl/fuse/fusefs.go

```diff
@@ -204,14 +221,43 @@ type inode struct {
 	kernfs.InodeDirectoryNoNewChildren
 	kernfs.OrderedChildren
 
-	locks vfs.FileLocks
-
+	NodeID uint64
 	dentry kernfs.Dentry
+	locks  vfs.FileLocks
+
+	// the owning filesystem. fs is immutable.
+	fs *filesystem
+
+	// size of the file.
+	size uint64
+
+	// attributeVersion is the version of inode's attributes.
+	attributeVersion uint64
+
+	// attributeTime is the remaining vaild time of attributes.
+	attributeTime uint64
+
+	// version of the inode.
+	version uint64
+
+	// link is result of following a symbolic link.
```

Comment:
"result" -> "resolved path"?

Better to explicit say if it's a cache. If so, please add the "FUSE_CACHE_SYMLINKS" flag to fuse init flags

Reply:
changed comment message
As for cache of the link, I believe it is the default behavior of Linux (https://elixir.bootlin.com/linux/v4.4/source/fs/namei.c#L4508) that's not specified by any flag, so we should probably do the same.

Reply:
What do you mean by "not specified by any flag"? Since it's default, so the `FUSE_CACHE_SYMLINKS` is also [used by default](https://elixir.bootlin.com/linux/v5.9-rc2/source/fs/fuse/inode.c#L1005).

Reply:
If Linux doesn't have a flag to turn this caching behavior on and off, is there a specific reason why we should do it differently?

Reply:
That flag is exactly what to turn it on. It's used in FUSE_INIT

Reply:
Unless you are saying 4.4 kernel does not have it. In that case, we should remove all the FUSE_INIT flags that are added after 4.4

Reply:
Yeah, I'm referring to 4.4 kernel. I'm not sure about FUSE_INIT, but feel in general we should follow 4.4 since that's what we are targeting